internal/app/issue_match/issue_match_handler_events.go

```diff
@@ -78,3 +83,204 @@ type RemoveEvidenceFromIssueMatchEvent struct {
 func (e *RemoveEvidenceFromIssueMatchEvent) Name() event.EventName {
 	return RemoveEvidenceFromIssueMatchEventName
 }
+
+func OnComponentInstanceCreate(db database.Database, event event.Event) {
```

Comment:
Isnt that the next feature? Issue Mapping/Matching?

Reply:
I've "spined off" this branch from `basic_matching`. I think this function was already there before.